Database/Updates/World/04-06-20-2017-alterPostionTable-stored.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE `ace_world`.`ace_position`
+  DROP COLUMN `landblock` ,
+  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
```

Comment:
you can test with the following:

ALTER TABLE `ace_world`.`ace_position`   
  DROP INDEX `idx_landblock`,
  DROP COLUMN `landblock` ,
  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) VIRTUAL,
  ADD  KEY `idx_landblock` (`landblock`);

select * from ace_position order by landblock

then 

ALTER TABLE `ace_world`.`ace_position`   
  DROP INDEX `idx_landblock`,
  DROP COLUMN `landblock` ,
  ADD COLUMN `landblock` INT(5) UNSIGNED AS ( ((landblockraw >> 16) & 0xFFFF)) STORED,
  ADD  KEY `idx_landblock` (`landblock`);

select * from ace_position order by landblock

it would be easy to just drop this column during import of PCAPS as well then add at the end of the import.   it takes 22 seconds to switch the column storage and build the index.